vms/avm/environment_test.go

```diff
@@ -230,6 +217,14 @@ func setup(tb testing.TB, c *envConfig) *environment {
 	}
 
 	require.NoError(vm.SetState(context.Background(), snow.NormalOp))
+
+	tb.Cleanup(func() {
```

Comment:
lock is acquired above. Each UT using setup is expected to eventually Unlock to that tb.Cleanup works properly